send-data/react.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: "Send data from client-side React apps to Axiom"
+description: "This page explains how to send data from your client-side React apps to Axiom using the @axiomhq/react library."
+overview: "The library for web and native user interfaces"
+sidebarTitle: React
+tags:
+  [
+    "js",
+    "axiom documentation",
+    "documentation",
+    "axiom",
+    "guide",
+    "logs",
+    "nextjs",
+    "next.js",
+    "next",
+    "next-axiom",
+    "@axiomhq/nextjs",
+    "javascript",
+  ]
+isPopular: true
+popularityOrder: 9
+logoId: "react"
+---
+
+## Installation
+
+```sh
+npm install --save @axiomhq/react @axiomhq/logging 
```

Comment:
lets make `@axiomhq/logging` come before the react package, to make it clear that this is as a dependency for the react lib.